go-controller/pkg/ovn/base_network_controller_multicast.go

```diff
@@ -30,10 +29,17 @@ const (
 	legacyMulticastDefaultDenyPortGroup = "mcastPortGroupDeny"
 )
 
-func getACLMatchAF(ipv4Match, ipv6Match string) string {
-	if config.IPv4Mode && config.IPv6Mode {
+func (bnc *BaseNetworkController) GetMulticastPortGroupName(base string) string {
```

Comment:
I guess that should be getClusterPortGroupName? Because cluster port groups are not only used by multicast, but e.g. by egress firewall
 If so, I would suggest to move it to base_network_controller, update the name and add a comment for which base names it should be used